pyproject.toml

```diff
@@ -1,12 +1,81 @@
 [build-system]
-build-backend = "setuptools.build_meta"
 requires = [
-    "setuptools >= 46.6.0",
+    "setuptools",
+    "lsst-versions >= 1.3.0",
     "wheel",
     "pybind11 >= 2.5.0",
     "numpy >= 1.18",
```

Comment:
Is this because we need the numpy C header files for the build with pybind11?

Reply:
Yes.